src/node_task_runner.cc

```diff
@@ -81,16 +68,48 @@ ProcessRunner::ProcessRunner(
   for (int i = 0; i < env_count; i++) {
     std::string name = env_items[i].name;
     std::string value = env_items[i].value;
+
+#ifdef _WIN32
+    // We use comspec environment variable to find cmd.exe path on Windows
+    // Example: 'C:\\Windows\\system32\\cmd.exe'
+    // If we don't find it, we fallback to 'cmd.exe' for Windows
+    if (name.size() == 7 && StringEqualNoCaseN(name.c_str(), "comspec", 7)) {
+      options_.file = value.c_str();
+    }
+#endif
+
     // Check if environment variable key is matching case-insensitive "path"
     if (name.size() == 4 && StringEqualNoCaseN(name.c_str(), "path", 4)) {
-      value.insert(0, current_bin_path + ":");
+      value.insert(0, current_bin_path);
     }
+
     // Environment variables should be in "KEY=value" format
     value.insert(0, name + "=");
     env_vars_.push_back(value);
   }
   uv_os_free_environ(env_items, env_count);
 
+#ifdef _WIN32
+  if (file.find("cmd.exe") != std::string::npos) {
+    // /d /s /c are only used for cmd.exe
+    command_args_ = {
+        options_.file, "/d", "/s", "/c", "\"" + command_str + "\""};
```

Comment:
Ok. For people who are idiots like myself, maybe add something like this:

```
// 
// /c  Carries out the command and exit.
// /d	Disables execution of AutoRun commands.
// /s Strip the first and last quotes (") around the <string> but leaves the rest of the command unchanged.
```

Reply:
Updated! You're definitely right.